cypress-tests/cypress/constants/selectors/manageGroups.js

```diff
@@ -9,22 +9,42 @@ export const groupsSelector = {
   addNewGroupModalTitle: '[data-cy="create-new-group-title"]',
   groupNameInput: "[data-cy=group-name-input]",
   cancelButton: "[data-cy=cancel-button]",
+  workspaceVarCreateLabel: '[data-cy="workspace-variable-create-label"]',
   createGroupButton: "[data-cy=create-group-button]",
+  adminListItem: '[data-cy="admin-list-item"]',
+  adminTitle: '[data-cy="admin-title"]',
+  adminAccessHelperText: '[data-cy="helper-text-admin-app-access"]',
+  appCreateHelperText: '[data-cy="app-create-helper-text"]',
+  appDeleteHelperText: '[data-cy="app-delete-helper-text"]',
+  resourceHeader: '[data-cy="resource-header"]:eq(1)',
```

Comment:
Use the existing selectors. For child components, add `eq()` in the code block instead of embedding it within the selectors.